### DISCUSSION TITLE
part-ii/2005-0/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#143] [Question comments]
Part II 2005 Number Theory: 1.I.1H 

https://questions.tripos.org/part-ii/2005-0/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

This is all bookwork except for the evaluation of the Legendre symbol. Here, we have

$$
\begin{align*}
(91/167) &= (7/167)(13/167) \\
&= -(167/7)(167/13) \\
&= -(6/7)(11/13) \\
&= -(-1)(-1) \\
&= -1 \\
\end{align*}
$$

</details>
